### ISSUE TITLE
Menu not found after reload

### ISSUE THREAD

**Issue by SaltyAimbOtter** [BetonQuest/RPGMenu#9]
## Bug report

**Spigot version:** `<!-- put your servers spigot version here -->`
This server is running Paper version git-Paper-474 (MC: 1.13.2) (Implementing API version 1.13.2-R0.1-SNAPSHOT)
**RPGMenu version:** `<!-- put your RPGMenu version here -->`
 RPGMenu version 3.0
**BetonQuest version:** `<!-- put your BetonQuest version here -->`
BetonQuest version 2.0-preview.13
### Description:
Menu does not work after /q reload with no error message or no change at all.

```
<!--if you have errors in console include them here,
[RPGMenu] Could not open menu ashes-Status.status: Unknown menu
```

### How to reproduce:
1. Create Menu
2. Use it ingame (open with command)
3. Change menu and reload BQ
4. /var does not work anymore
https://youtu.be/2KYQe6L6EvQ

### Checklist:

- [X] I included all versions (use `/version` command)
- [X] I made sure there aren't duplicates of this report [(use Search)](https://github.com/joblo2213/RPGMenu/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3ABug+)
- [X] I made sure I am using an up-to-date version of RPGMenu
- [X] I made sure the bug/error is not caused by any other plugin

**Comment by Momonche** [comment]
I have the same problem
+1

**Comment by Punishkey** [comment]
mee to

edit: After two hours and change all yml work with reload. I dont know what has really changed

**Comment by bundabrg** [comment]
Do you also need to /rpgmenus reload?

**Comment by SaltyAimbOtter** [comment]
No, it looks for BQ reload and then reloads with it.

**Comment by bundabrg** [comment]
Cannot reproduce. My steps:
1. Menu file as follows:

```YAML
title: "&5&lThe Menu"
command: /menu
height: 6
slots:
  0: Glass
  1: Glass
items:
  Glass:
    item: Glass
    amount: 1
    text:
  Dirt:
    item: Dirt
    amount: 5
    text:
```

2. /q reload, and /menu shows the menu with 2 glass items
3. Edit file as follows:

```YAML
title: "&5&lThe Menu"
command: /menu
height: 6
slots:
  0: Dirt
  1: Glass
items:
  Glass:
    item: Glass
    amount: 1
    text:
  Dirt:
    item: Dirt
    amount: 5
    text:
```

4. /q reload, /menu shows menu with 5dirt and 1 glass

**Comment by SaltyAimbOtter** [comment]
It did not appear here again too. Maybe @bundabrg fixed it by accident with his new BQ build. Or it´s a bug that only appears under certain circumstances.

**Comment by Momonche** [comment]
I still have the problem

[22:22:18 INFO]: MOMO3915 issued server command: /q reload
[22:22:19 INFO]: [BetonQuest] There are 843 conditions, 2499 events, 606 objectives and 54 conversations loaded from 35 packages.
[22:22:19 INFO]: [RPGMenu] Reloaded 9 menus
[22:22:21 INFO]: MOMO3915 issued server command: /quest
[22:22:21 INFO]: [RPGMenu] Could not open menu default.QueteEnCours: Unknown menu

But I don't known how it happening

**Comment by Momonche** [comment]
Hello !
I think this bug was fixed on bundabrg's builds but with the build find on Discord that issue reappear

It's very horrible because I can't edit quest on my production server

**Comment by SaltyAimbOtter** [comment]
Oh thanks for letting me know, I must have forked the wrong one!
I will push an update to this soonish since we have a repo again (which prevented me from doing so eariler).

**Comment by seyfahni** [comment]
This should be fixed with RPGMenu 4.0.3, could you please try upgrading?
